sys-fs/python-btrfs/files/13-006-btrfs-search-metadata-use-FileSystem-block_groups.patch

```diff
@@ -0,0 +1,32 @@
+From 59c8daca3a05f86001b1dc36b2ab2e5961bf7bc0 Mon Sep 17 00:00:00 2001
```

Comment:
Links at the top of each patch giving the source pls? (commit links)

Reply:
Argh! Of course, will add.